monitor/census.go

```diff
@@ -1930,6 +1961,11 @@ func (cen *censusMetricsCounter) recordAIJobPricePerUnit(pipeline string, Model
 		cen.mAIRequestPrice.M(pricePerUnit)); err != nil {
 		glog.Errorf("Error recording metrics err=%q", err)
 	}
+	SendQueueEvent("ai_request_price_per_unit", map[string]string{
```

Comment:
I somehow feel that we have too many event types. It's up to @ecmulli , but I believe that we should not send "every event we can think of", but rather limit to what we need and what makes sense for the analytics. Maybe even you can both discuss and create a table with the event types that would make sense in the context of Gateway.

Reply:
@ecmulli let's remove entirely the events from the census from this PR, I added them before the payments during the local testing phase, but now we want the payments data to get out in staging, so this addressed both the comments on the events and we remove the logic from census. Just made the commit, so that we're sending data exclusively on payment gen